boilerplate/app/navigators/AppNavigator.tsx

```diff
@@ -36,7 +36,9 @@ import { navigationRef, useBackButtonHandler } from "./navigation-utilities"
 export type AppStackParamList = {
   Welcome: undefined
   Login: undefined
+  // @demo remove-block-start
   Demo: NavigatorScreenParams<DemoTabParamList>
+  // @demo remove-block-end
```

Comment:
Added these blocks comments to test the main `src/commands/remove-demo.ts` command. Not the best use, will need to spend some more time rethinking this.

We should either:
- remove references to `@demo remove-*` in the boilerplate
- do a more in-depth test of what we want to remove